provd/persist/util.py

```diff
@@ -552,3 +551,22 @@ def __init__(self, collection):
 
     def __getattr__(self, name):
         return getattr(self._collection, name)
+
+
+def _new_key_fun_from_key(key):
+    """
+    Return a function usable for the key parameter of the sorted function
+    from a [sort] key
+    """
+    split_key = key.split('.')
+
+    def func(document):
+        cur_elem = document
+        try:
+            for cur_key in split_key:
+                cur_elem = cur_elem.get(cur_key, '') or ''
+        except AttributeError as e:
```

Comment:
You will never get an `AttributeError` with `get` and a dict

Reply:
if a string, I will have AttributeError :)